server/subscribe.go

```diff
@@ -320,29 +321,19 @@ func httpSubscribeWebhook(p *Plugin, w http.ResponseWriter, r *http.Request) (in
 		return http.StatusForbidden, fmt.Errorf("Request URL: secret did not match")
 	}
 
-	wh, jwh, err := ParseWebhook(r.Body)
+	bb, err := ioutil.ReadAll(r.Body)
```

Comment:
This is some copying. Could we just pass the reader?

Reply:
Afaik, if we pass the reader, we could be keeping a pointer to a reader (and the underlying socket/whatever is backing it?) open until we get around to reading it. That might have weird unanticipated effects. I'd rather read the data and then return from the httpserve, then deal with the data sometime/somewhere else.

Reply:
Mmmm yes it would prevent the underlying system from reusing the connection. To get the full benefits of this I guess we should copy. Hopefully we don't end up shooting ourselves in the foot with memory pressure :P